CMakeLists.txt

```diff
@@ -98,6 +101,7 @@ add_subdirectory(tools)
 # benchmark that requires bpftime libraries
 add_subdirectory(benchmark)
 
+
 set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>DLL")
 
-install(TARGETS bpftime-agent bpftime-agent-transformer bpftime-syscall-server CONFIGURATIONS Release Debug RelWithDebInfo DESTINATION ~/.bpftime)
+install(TARGETS bpftime-agent bpftime_text_segment_transformer bpftime-syscall-server CONFIGURATIONS Release Debug RelWithDebInfo DESTINATION ~/.bpftime)
```

Comment:
is it possible not change the install name?

Reply:
This is the target name. The file name was kept the same as before, see https://github.com/Officeyutong/bpftime/blob/8526a42652bca7ecadd9df5525003c4307a5ebfa/attach/text_segment_transformer/CMakeLists.txt#L6